github/resource_github_repository.go

```diff
@@ -137,6 +142,7 @@ func resourceGithubRepositoryObject(d *schema.ResourceData) *github.Repository {
 	licenseTemplate := d.Get("license_template").(string)
 	gitIgnoreTemplate := d.Get("gitignore_template").(string)
 	archived := d.Get("archived").(bool)
+	topics := d.Get("topics").([]string)
```

Comment:
This will come out of `ResourceData` as `[]interface{}`, so you will need to write a simple func to "expand" this state to the SDK's representation:

```go
func expandStringList(configured []interface{}) []string {
	if len(configured) == 0 {
		return nil
	}
	vs := make([]string, 0, len(configured))
	for _, v := range configured {
		val, ok := v.(string)
		if ok && val != "" {
			vs = append(vs, val)
		}
	}
	return vs
}
```